crates/rust-analyzer/src/cli/analysis_stats.rs

```diff
@@ -256,6 +259,8 @@ impl flags::AnalysisStats {
         if let Some(instructions) = total_span.instructions {
             report_metric("total instructions", instructions, "#instr");
         }
+        let db = host.raw_database_mut();
+        db.trigger_lru_eviction();
```

Comment:
I think this needs to happen before the `let total_span = analysis_sw.elapsed()` line :sweat_smile:

Reply:
whoops! good point.